package.json

```diff
@@ -23,6 +24,7 @@
     "lint:es": "eslint src/**/*.{js,vue}",
     "lint:style": "vue-cli-service lint:style",
     "serve:storybook": "cd ./storybook && yarn run serve",
+    "prepare": "echo hello lifecycle hook",
```

Comment:
just making sure this change is intended here

Reply:
I'll remove that. Just testing.